src/VisualStudio/Core/Def/Microsoft.VisualStudio.LanguageServices.csproj

```diff
@@ -83,6 +83,7 @@
     <Reference Include="System.ComponentModel.Composition" />
   </ItemGroup>
   <ItemGroup>
+    <InternalsVisibleTo Include="Microsoft.VisualStudio.TestWindow.CodeLens" Key="$(UnitTestingKey)" />
```

Comment:
❗️ This IVT cannot be added. I can help provide an alternative approach via the external access layer.

Reply:
@sharwell I just synced with @jinujoseph @mavasani and I thought the idea was to update all IVTs with additional metadata (namespaces consumed) and that in the meantime it was ok to add direct IVTs. We'd rather not use the external access layer because we want to avoid additional assembly loads for this scenario. Did I get something wrong?

cc @ManishJayaswal

Reply:
We will need to wait until the analyzer is ready for use, or use the external access layer in the interim time. Unfortunately the process for eliminating IVTs once added is tremendously more complicated than expected, entirely due to causes outside our control. The only remaining option we have left is to disallow their creation.